tools/random_sample_cohort.py

```diff
@@ -0,0 +1,99 @@
+import sys
+from random import sample
+
+import pandas as pd
+from numpy import full
+
+HEADER_DISTINCT_ID = '$distinct_id'
+HEADER_FIRST_SEEN = '$mp_first_event_time'
+HEADER_COHORT = 'Cohort'
+
+RANDOM_SAMPLE_PER_DAY_SIZE = 20
```

Comment:
I think we should avoid creating any difference between different cohorts if we can manage it. The distribution might be the same per month, but the parameters to that distribution will be different between cohorts (if the sizes per day are different, sample variance would be different, for example). Let's not overoptimize things (and I also think that a cohort of 560 people will be big enough!).

20 is the largest number we can reasonable support - if we grow a lot in the future and in this is bigger, we can switch all future cohorts to that. We want to keep them as similar as possible otherwise.